src/hooks.jl

```diff
@@ -255,7 +255,8 @@ function core_parser_hook(code, filename::String, lineno::Int, offset::Int, opti
                     #-#-#-
                     """)
         end
-        @error("JuliaSyntax parser failed — falling back to flisp!",
+        @error("""JuliaSyntax parser failed — falling back to flisp!
+                  This is probably not your fault, please submit a bug report (https://github.com/JuliaLang/JuliaSyntax.jl/issues)""",
```

Comment:
```suggestion
                  This is not your fault. Please submit a bug report to https://github.com/JuliaLang/JuliaSyntax.jl/issues""",
```
I initially thought that a "... and include the string that caused the failure" would be a good idea, but that's part of the issue template.